charts/stable/shiori/values.yaml

```diff
@@ -1,27 +1,11 @@
 image:
-  repository: tccr.io/truecharts/shiori
```

Comment:
please do not do this and only apply container source changes on container repo...

Reply:
im waiting on the image container change first.